sdk/core/src/mock_transport.rs

```diff
@@ -0,0 +1,78 @@
+#[cfg(any(
+    feature = "mock_transport_generate",
+    feature = "mock_transport_consume"
+))]
+const ENV_TRANSACTION_NUMBER: &str = "AzureSDKforRustTransactionNumber";
```

Comment:
As mentioned earlier, these should be hard-coded / calculated.

Reply:
We changed the path to be hardcoded but the operation counter in a transaction is variable by nature. I'm open to suggestions on how to do this differently but so far that's the only option I've found.